scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -23,19 +23,91 @@ source "$(dirname "${0}")"/../functions
 L_PWD=$(pwd)
 cd "${DEVON_IDE_HOME}" || exit 255
 
+# Install Eclipse plugins by feature group id
 # $1: plugin feature group ID
 # $2: plugin URL (update site)
 function doInstallEclipsePlugin() {
-  local plugin="${1/\.feature\.group*/}"
-  echo "Installing eclipse plugin ${plugin} from ${2}"
-  # shellcheck disable=SC2086
-  "${ECLIPSE_HOME}/eclipse" -nosplash -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.equinox.p2.director -repository "${2}" -installIU "${1}" -vmargs ${ECLIPSE_VMARGS}
-  result=${?}
-  if [ "${result}" != 0 ]
+  # If plugin_name argument ${3} is passed
+  if [ -n "${3}" ]
   then
-    doEchoAttention "Installation of plugin ${plugin} failed. Please try to install manually or rerun setup (devon eclipse setup)."
+    echo "Installing Eclipse plugin '${3}' from URL ${2}"
+  else
+    echo "Installing Eclipse plugin '${1}' from URL ${2}"
   fi
-  return "${result}"
+
+  if [ -z "${1}" ]
+  then
+    doEcho "No plugins to install."
+    return
+  fi
+
+  local plugin_dependencies
+  plugin_dependencies=$(doCheckPluginsInstalled "${1}")
+
+  # Install plugins and write debug info into temp file
+  local debug_info_file
+  debug_info_file=$(mktemp)
+  "${ECLIPSE_HOME}/eclipsec" -consoleLog -nosplash -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.equinox.p2.director -repository "${2}" -installIU "${1}" -vmargs "${ECLIPSE_VMARGS}" > "${debug_info_file}" 2>&1
+  local exit_code=$?
+  # Write only lines with !MESSAGE into debug info
+  local debug_info
+  debug_info=$(grep -E "!MESSAGE" "${debug_info_file}")
+  
+  rm "${debug_info_file}"
+
+  if [[ "${debug_info}" == *"Overall install request is satisfiable"* ]] && [ "${exit_code}" == 0 ]; then
```

Comment:
now we could grep this on the log file (before deleting it - do we have to delete the log after all?) to avoid reading it into memory.

Reply:
I've made the suggested changes and pushed them